### PR TITLE
chapter_11: account activation

### PR DESCRIPTION
**Pull request by quanghuy74**
image
chapter_11: account acticvation

### CODE REVIEW

**Comment by quanghuy74** [comment]
ready

File: app/models/user.rb [hunk 1]
```diff
@@ -1,20 +1,47 @@
...
+
+  def authenticated? attribute, token
+    digest = send "#{attribute}_digest"
+    return false unless digest
+
+    BCrypt::Password.new(digest).is_password? token
+  end
+
+  def activate
+    update_column activated: true, activated_at: Time.zone.now
```
Comment (vanvtt-0952) [inline app/models/user.rb]: `update_columns` chứ chú, chạy thử chưa (tat)

**Comment by quanghuy74** [comment]
ready

**Comment by vanvtt-0952** [comment]
merged

**Pull request merged**